tests/lib/SimpleSAML/Utils/AttributesTest.php

```diff
@@ -117,10 +112,8 @@ public function testGetExpectedAttributeMultipleValues()
             ],
         ];
         $expected = 'attribute';
-        $this->expectException(
-            \SimpleSAML\Error\Exception::class,
-            'More than one value found for the attribute, multiple values not allowed.'
-        );
+        $this->expectException(\SimpleSAML\Error\Exception::class);
+        $this->expectExceptionMessage('More than one value found for the attribute, multiple values not allowed.');
         Attributes::getExpectedAttribute($attributes, $expected);
```

Comment:
Not sure in what PHPunit-version this was allowed, but today it's split in two calls (actually for years already)